### PR TITLE
Add debug logs when the Kyverno test fails

### PR DESCRIPTION
**Pull request by mprahl**
This should help debug issues such as:
https://github.com/stolostron/backlog/issues/24699

Closes https://github.com/stolostron/backlog/issues/24699

### CODE REVIEW

File: test/common/common.go [hunk 1]
```diff
@@ -258,3 +258,11 @@ func DoRootComplianceTest(hub dynamic.Interface, policyName string, compliance p
 		1,
 	).Should(gomega.Equal(compliance))
 }
+
+func OutputDebugInfo(testName string) {
+	ginkgo.GinkgoWriter.Printf("%s test Kubernetes info:\n", testName)
+
+	for _, resource := range []string{"policies", "placementrule", "placement", "placementbinding"} {
```
Comment (JustinKuli) [inline test/common/common.go:265]: I think we should qualify the "policies" name - do you want kyverno policies, open-cluster-management policies, maybe both?

**Review by JustinKuli — CHANGES_REQUESTED** [review]

File: test/common/common.go [hunk 2]
```diff
@@ -258,3 +258,19 @@ func DoRootComplianceTest(hub dynamic.Interface, policyName string, compliance p
 		1,
 	).Should(gomega.Equal(compliance))
 }
+
+func OutputDebugInfo(testName string, additionalResources []string) {
```
Comment (JustinKuli) [inline test/common/common.go:262]: Nit: could be variadic for the additionalResources - currently to not get any additional resources, you need to pass `nil`. But this is totally not necessary right now.

**Review by JustinKuli — DISMISSED** [review]

File: test/common/common.go (hunk 2 above)
Comment (mprahl) [reply test/common/common.go]: That's a nice suggestion!

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**